sweat/algorithms/metrics/power.py

```diff
@@ -0,0 +1,71 @@
+import pandas as pd
+from sweat.utils import cast_array_to_original_type
+
+
+def wpk(power, weight):
+    """Watts per kilo
+
+    Parameters
+    ----------
+    power : list, ndarray, series
+    weight : number
+
+    Returns
+    -------
+    array-like
+    """
+
+    if not isinstance(power, pd.Series):
```

Comment:
Let's see if I can come up with a satisfactory solution in my "on the fly decorator type casting poc" so we can come to solution for this in #5.

Reply:
agree